extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcUtils.java

```diff
@@ -57,15 +61,16 @@ private static Object findClaimValue(String claimPath, JsonObject json, String[]
         Object claimValue = json.getValue(pathArray[step]);
         if (claimValue == null) {
             if (mustExist) {
-                throw new OIDCException(
-                        "No claim exists at the path " + claimPath + " at the path segment " + pathArray[step]);
+                LOG.warn("No claim exists at the path " + claimPath + " at the path segment " + pathArray[step]);
+                throw new OIDCException();
             }
         } else if (step + 1 < pathArray.length) {
             if (claimValue instanceof JsonObject) {
                 int nextStep = step + 1;
                 return findClaimValue(claimPath, (JsonObject) claimValue, pathArray, nextStep, mustExist);
             } else {
-                throw new OIDCException("Claim value at the path " + claimPath + " is not a json object");
+                LOG.warn("Claim value at the path " + claimPath + " is not a json object");
+                throw new OIDCException();
```

Comment:
Could you explain why you chose to do that this way? I.e. going from an exception message to a warn?

Reply:
@gsmet Please see my response to Loic, at the moment it is not possible to wrap the OIDC exception. I'll make it the error message again once `ForbiddenException`/etc are updated, does it sound OK ?

Reply:
I'll open an issue to track it so as I don't forget

Reply:
I'm going to do a small PR to the quarkus-security and will follow up with the updates to this code but also to other Quarkus code where the causes are not included for those exceptions

Reply:
FYI https://github.com/quarkusio/quarkus/pull/5952